apps/docs-website/package.json

```diff
@@ -32,7 +32,8 @@
     "react-debounce-input": "^3.3.0",
     "react-dom": "^18.3.1",
     "react-hook-form": "^7.52.1",
-    "react-select": "^5.7.4"
+    "react-select": "^5.7.4",
+    "search-insights": "^2.17.3"
```

Comment:
why this dependency? I don't think we need this

Reply:
It's an agolia package, missing peer dependency (it was auto-installed, but auto-installing means that the fact that this is installed is hidden, so I turned it off).